pkg/limits/ingest_limits.go

```diff
@@ -157,7 +159,7 @@ func NewIngestLimits(cfg Config, lims Limits, logger log.Logger, reg prometheus.
 	kCfg.AutoCreateTopicEnabled = true
 	kCfg.AutoCreateTopicDefaultPartitions = cfg.NumPartitions
 
-	s.client, err = client.NewReaderClient("ingest-limits", kCfg, logger, reg,
+	s.reader, err = client.NewReaderClient("ingest-limits-reader", kCfg, logger, reg,
```

Comment:
Just FYI this will change our metrics as `ingest-limits` is used as a prefix for all Kafka related metrics. I will fix this though in a follow up PR as we will use the same client for producing and consuming.

Reply:
Yes please sir. This is a stopgap to avoid double metrics register panics.